processes/presenting-at-conferences.rst

```diff
@@ -0,0 +1,154 @@
+#########################
+Presenting at Conferences
+#########################
+
+This is our playbook for presenting on behalf of DM at both project and public meetings.
+
+.. seealso::
+
+   Conference proceedings papers are coordinated by the LSST Project Publication Board.
+   See :doc:`publication-policy` for more information.
+
+.. _presenting-interaction-report:
+
+Interaction report
+==================
+
+As DM team members, we work within an NSF and DOE funded MREFC construction project.
+It's important that we're consistent with the Project when we speak in public and with government officials.
+As you're preparing your presentation:
+
+- `File an interaction report <https://project.lsst.org/interactions>`__. Interaction reports are required when presenting at a public meeting or with government officials.
+- Use and refer to the Project's `pre-approved material <https://project.lsst.org/preapproved>`__ for presentations, including schedule information.
+- Use the Project's `Key Numbers <https://confluence.lsstcorp.org/display/LKB/LSST+Key+Numbers>`__.
+
```

Comment:
...and while we're taking about interaction reports, what about the [Speakers Bureau](https://project.lsst.org/speakersbureau/)? Are we supposed to interact with that? It sounds all very important, but doesn't seem to have been used [in some time](https://project.lsst.org/speakersbureau/all-events).